README.md

```diff
@@ -46,13 +47,13 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Not deployed yet...]()
+- Not deployed yet.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 📺 Presentation <a name="video-link"></a>
 
-[Not developed yet...]()
+- Not developed yet.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] As the previous reviewer suggested that please remove the Live Demo and Presentation section since you have nothing in these sections :fearful: You can add these sections again once you have required data for these sections :+1: